src/main/kotlin/interpreter/functions/PyBuiltinFunction.kt

```diff
@@ -54,10 +54,11 @@ abstract class PyBuiltinFunction(val name: String) : PyFunction(PyBuiltinFunctio
         }
     }
 
-    override fun pyStr(): PyString =
```

Comment:
No. Everything that's exposed as a python function, is also a Kotlin function. These are the impls of `__str__` and `__repr__`, and as such are exposed as functions.